Cargo.toml

```diff
@@ -41,7 +41,7 @@ homepage = "https://spice.ai"
 license = "Apache-2.0"
 repository = "https://github.com/spiceai/spiceai"
 rust-version = "1.84"
-version = "1.1.0"
+version = "1.1.0-unstable"
```

Comment:
Alternatively this can be marked as development version - `1.1.0-dev`

Reply:
Anyone can build from trunk, not just for development purposes. The connotation is that doing so is possible, but its inherently `unstable` compared to using a released binary.